lab_1/start.py

```diff
@@ -2,17 +2,18 @@
 Concordance implementation starter
 """
 
-from main import read_from_file
 import os
+import main

```

Comment:
Usually, the import statement goes like this:
`from main import func1, func2, func4 ...` unless you want to import all functions from the module and use them.